release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1145,6 +1145,50 @@ With this update, the Ingress Operator no longer adds the `clientca-configmap` f
 [id="ocp-4-15-openshift-cli-bug-fixes"]
 ==== OpenShift CLI (oc)
 
+[discrete]
+[id="ocp-4-15-openshift-oc-mirror-bug-fixes"]
+==== oc-mirror
+
+Prior to this release, if a user wanted to filter operator packages by channel, that is, `mirror.operators.catalog.packages.channels`, they were required to include the default channel for the package, even if they had no intention of using bundles from that channel.  Based on this information, if the package had a default channel that was not part of the catalog, the catalog is considered invalid.
```

Comment:
```suggestion
Prior to this release, if a user wanted to filter operator packages by channel, that is, `mirror.operators.catalog.packages.channels`, they were required to include the default channel for the package in the imageSetConfig, even if they had no intention of using bundles from that channel.  This increased the volume of mirrored images, but kept the default channel intact in the catalog, which guarantied better upgrades for the operator.
```
the default channel is in the catalog, we' re asking the user to include it in the imagesetconfig. I think this is the confusing part. 
So we cannot use the statement :
```
Based on this information, if the package had a default channel that was not part of the catalog, the catalog is considered invalid.
```
Or we should say:
```
Based on this information, if the imageSetConfig did not include the default channel for the package,  the resulting catalog is considered invalid.
```

Reply:
Ack.